src.common/DataStructures/Grid2D.ts

```diff
@@ -1,39 +1,30 @@
-export class Grid2D<T>
-{
+export class Grid2D<T> {
```

Comment:
Where we're using the T type generic here, is it because we plan for this to accept multiple types, or is it just a placeholder for an object we might make later?

Reply:
One aspect is that all classes in `src.common` will be removed at some point once I finish the external libraries and publish them.
Second aspect is we currently only use it for the floor tiles, but soon enough we'll want to have a grid of entities to avoid having to go through the list of all of them in level just to see if there isn't anything on a tile something wants to walk on.